cmd/otelcol/Dockerfile.windows

```diff
@@ -27,6 +27,25 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${env:JMX_METRIC_GATHERER_RELEASE}/opentelemetry-jmx-metrics.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JDK_VERSION="11.0.20_8"
+ARG OPENJDK_BASE_URL="https://github.com/adoptium/temurin11-binaries/releases/download"
+ENV JAVA_HOME="C:\java\jre"
+ENV JAVA_PATH="$JAVA_HOME\bin"
+RUN Set-Variable -Name "ENCODED_VER" -Value "\"$env:JDK_VERSION\".replace(\"_\", \"%2B\")"; \
+    Invoke-WebRequest -Uri "${env:OPENJDK_BASE_URL}/jdk-${ENCODED_VER}/OpenJDK11U-jre_x64_windows_hotspot_${env:JDK_VERSION}.zip" -OutFile "openjdk_jre.zip" ; \
+    Expand-Archive "openjdk_jre.zip" -DestinationPath "C:\java" ;
+RUN Move-Item -Path C:\java\jdk-* -Destination ${Env:JAVA_HOME}
+RUN Remove-Item "openjdk_jre.zip" -force
+RUN setx /M PATH $(${Env:PATH} + \";${Env:JAVA_PATH}\")
+# update otel-java/autoconfigure defaults to prevent undesired telemetry
+RUN setx /M OTEL_TRACES_EXPORTER "none"
+RUN setx /M OTEL_LOGS_EXPORTER "none"
```

Comment:
Should we add some basic tests to verify the paths and env vars to [CI](https://github.com/signalfx/splunk-otel-collector/blob/main/.github/workflows/win-package-test.yml#L362)?

Reply:
given https://github.com/signalfx/splunk-otel-collector/pull/3262#discussion_r1307944696 what would you suggest for the path check?

Reply:
For now, maybe we can just try to execute something like `java -version` in the container?